src/functions.php

```diff
@@ -766,17 +766,15 @@ function _parse_message($message)
     }
 
     $rawHeaders = substr($message, 0, $headerDelimiterPosition + 2); // We preserve the last \r\n, hence +2
-
```

Comment:
If you make `\r` optional, you have to change this code as well.

Reply:
Argh, this again (nice catch).

This thing is starting to annoy me. There aren't even any tests that would verify that this works.